astroid/raw_building.py

```diff
@@ -319,65 +319,73 @@ def ast_from_function(func, built_objects, module, name=None, parent=None):
                   itertools.groupby(signature.parameters.values(),
                                     operator.attrgetter('kind'))}
 
-    def extract_args(parameters, parent):
-        '''Takes an iterator over Parameter objects and returns three
-        sequences, arg names, default values, and annotations.
-
-        '''
-        names = []
-        defaults = []
-        annotations = []
+    def _extract_args(parameters, parent):
+        """Generate an iterator of params from Python parameters."""
```

Comment:
Since we're now going to have two objects named Parameter (the astroid node and the inspect/funcsigs.Parameter), I'd rewrite this to say something like, "Generates an iterator of Parameter nodes from an iterator of inspect/funcsigs.Parameter objects."